### PR TITLE
feat: fix a settings screen issues for a design review  [PART 2] - ACC - 252

### PR DESCRIPTION
**Pull request by BrzozowskiPawel**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?
ADD ACCOUNT AND SETIINGS SHOULD BE SEMIBOLD. 
- **FIXED** (bold font applied).
KEY FINGERPRINT CELL SHOULD BE GREY OR WHITE?
 - White, **FIXED**. After a talk with Anette this cell should be white.
Changed uppercased text to capitalised.

#### Test Coverage (Optional)

- [x] I have added automated test to this contribution

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by BrzozowskiPawel** [comment]
After research, I found that uppercased text in **uitableviewheaderfooterview** is done by the default -  source [link](https://stackoverflow.com/questions/61997261/is-there-a-function-to-set-the-section-title-of-the-tableview-so-that-the-title).  I have applied the feature that I was thinking of at the beginning.

File: Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/SettingsClientViewController.swift [hunk 1]
```diff
@@ -227,8 +227,8 @@ final class SettingsClientViewController: UIViewController,
         case .fingerprintAndVerify:
             if (indexPath as NSIndexPath).row == 0 {
                 if let cell = tableView.dequeueReusableCell(withIdentifier: FingerprintTableViewCell.zm_reuseIdentifier, for: indexPath) as? FingerprintTableViewCell {
-
                     cell.selectionStyle = .none
+                    cell.separatorInset = .zero
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/SettingsClientViewController.swift:231]: Why do we need this?

File: Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientListViewController.swift [hunk 2]
```diff
@@ -326,6 +326,7 @@ final class ClientListViewController: UIViewController,
     func tableView(_ tableView: UITableView, willDisplayHeaderView view: UIView, forSection section: Int) {
         if let headerFooterView = view as? UITableViewHeaderFooterView {
             headerFooterView.textLabel?.textColor = headerFooterViewTextColor
+            headerFooterView.textLabel?.text =  headerFooterView.textLabel?.text?.capitalized
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientListViewController.swift:329]: You can revert these changes, we will keep uppercase for now

File: Wire-iOS/Sources/UserInterface/Settings/SettingsTableViewController.swift [hunk 3]
```diff
@@ -260,6 +260,7 @@ final class SettingsTableViewController: SettingsBaseTableViewController {
     func tableView(_ tableView: UITableView, willDisplayHeaderView view: UIView, forSection section: Int) {
         if let headerFooterView = view as? UITableViewHeaderFooterView {
             headerFooterView.textLabel?.textColor = SemanticColors.Label.textSectionHeader
+            headerFooterView.textLabel?.text =  headerFooterView.textLabel?.text?.capitalized
```
[the remark this answers is not in the document]
Comment (BrzozowskiPawel) [reply Wire-iOS/Sources/UserInterface/Settings/SettingsTableViewController.swift:263]: Chnaged.
Ok sure if you say so I will revert it.

File: Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientListViewController.swift (hunk 2 above)
Comment (BrzozowskiPawel) [reply Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientListViewController.swift:329]: Yes. No problem I did it.

File: Wire-iOS/Resources/da.lproj/Localizable.strings [hunk 4]
```diff
@@ -884,7 +884,7 @@
 "profile.details.title" = "Detaljer";
 "profile.details.guest" = "Gæst";
 "profile.details.group_admin" = "Group admin";
-"profile.details.partner" = "external";
+"profile.details.partner" = "External";
```
Comment (KaterinaWire) [inline Wire-iOS/Resources/da.lproj/Localizable.strings:887]: Please do not change localizable strings. It will trigger this string to be translated to each language

File: Wire-iOS/Sources/UserInterface/Availability/AvailabilityStringBuilder.swift [hunk 5]
```diff
@@ -92,4 +92,5 @@ final class AvailabilityStringBuilder: NSObject {
             return IconColors.foregroundAvailabilityAway
         }
     }
+
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Availability/AvailabilityStringBuilder.swift:95]: Unnecessary changes

File: Wire-iOS/Sources/UserInterface/Availability/AvailabilityTitleView.swift [hunk 6]
```diff
@@ -153,3 +153,20 @@ final class AvailabilityTitleView: TitleView, Themeable, ZMUserObserver {
     }
 
 }
+
+struct AvailabilityColor {
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Availability/AvailabilityTitleView.swift:157]: Why do we need it?

File: Wire-iOS/Sources/UserInterface/Components/NSTextAttachment+Icon.swift [hunk 7]
```diff
@@ -25,18 +25,41 @@ extension NSTextAttachment {
...
+                               insets: UIEdgeInsets? = nil,
+                               borderWidth: CGFloat? = nil) -> NSTextAttachment {
+        var image: UIImage
         if let insets = insets {
             image = icon.makeImage(size: iconSize, color: color).with(insets: insets, backgroundColor: .clear)!
         } else {
             image = icon.makeImage(size: iconSize, color: color)
         }
 
+        if let borderWidth = borderWidth {
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Components/NSTextAttachment+Icon.swift:37]: You removed this in part 1, why do you keep it in this PR?

File: Wire-iOS/Sources/UserInterface/Components/NSTextAttachment+Icon.swift [hunk 8]
```diff
@@ -25,18 +25,41 @@ extension NSTextAttachment {
...
+
         let attachment = NSTextAttachment()
         attachment.image = image
         let ratio = image.size.width / image.size.height
         attachment.bounds = CGRect(x: 0, y: verticalCorrection, width: iconSize.rawValue * ratio, height: iconSize.rawValue)
         return attachment
     }
 }
+
+extension UIImage {
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Components/NSTextAttachment+Icon.swift:49]: And this

**Comment by BrzozowskiPawel** [comment]
@KaterinaWire you are right I see that there are some leftovers. 
I remember delting it with the merge. I will check it again.

**Comment by BrzozowskiPawel** [comment]
Delated leftovers and clean up after the merge.

**Review by ghuMustafa — APPROVED** [review]

**Review by KaterinaWire — APPROVED** [review]
